python/dgl/nn/pytorch/conv/gatv2conv.py

```diff
@@ -0,0 +1,291 @@
+            [[ 0.1908, -1.2095],
+            [-0.0129,  0.6408],
+            [-0.8135,  0.1157]],
+            [[ 0.0596, -0.8487],
+            [-0.5421,  0.4022],
+            [-0.4805,  0.1156]],
+            [[-0.0935, -0.4273],
+            [-1.1850,  0.1123],
+            [-0.2002,  0.1155]]], grad_fn=<GSpMMBackward>)
+    """
+    def __init__(self,
+                 in_feats,
+                 out_feats,
+                 num_heads,
+                 feat_drop=0.,
+                 attn_drop=0.,
+                 negative_slope=0.2,
+                 residual=False,
+                 activation=None,
+                 allow_zero_in_degree=False,
+                 bias=True,
+                 share_weights=False):
+        super(GATv2Conv, self).__init__()
+        self._num_heads = num_heads
+        self._in_src_feats, self._in_dst_feats = expand_as_pair(in_feats)
+        self._out_feats = out_feats
+        self._allow_zero_in_degree = allow_zero_in_degree
+        if isinstance(in_feats, tuple):
+            self.fc_src = nn.Linear(
+                self._in_src_feats, out_feats * num_heads, bias=bias)
+            self.fc_dst = nn.Linear(
+                self._in_dst_feats, out_feats * num_heads, bias=bias)
+        else:
+            self.fc_src = nn.Linear(
+                self._in_src_feats, out_feats * num_heads, bias=bias)
+            if share_weights:
+                self.fc_dst = self.fc_src
+            else:
+                self.fc_dst = nn.Linear(
+                    self._in_src_feats, out_feats * num_heads, bias=bias)
+        self.attn = nn.Parameter(th.FloatTensor(size=(1, num_heads, out_feats)))
+        self.feat_drop = nn.Dropout(feat_drop)
+        self.attn_drop = nn.Dropout(attn_drop)
+        self.leaky_relu = nn.LeakyReLU(negative_slope)
+        if residual:
+            if self._in_dst_feats != out_feats:
+                self.res_fc = nn.Linear(
+                    self._in_dst_feats, num_heads * out_feats, bias=bias)
+            else:
+                self.res_fc = Identity()
+        else:
+            self.register_buffer('res_fc', None)
+        self.reset_parameters()
+        self.activation = activation
+
+    def reset_parameters(self):
+        """
+        Description
+        -----------
+        Reinitialize learnable parameters.
+        Note
+        ----
+        The fc weights :math:`W^{(l)}` are initialized using Glorot uniform initialization.
+        The attention weights are using xavier initialization method.
+        """
+        gain = nn.init.calculate_gain('relu')
+        if hasattr(self, 'fc'):
+            nn.init.xavier_normal_(self.fc.weight, gain=gain)
+        else:
+            nn.init.xavier_normal_(self.fc_src.weight, gain=gain)
+            nn.init.xavier_normal_(self.fc_dst.weight, gain=gain)
+        nn.init.xavier_normal_(self.attn, gain=gain)
+        if isinstance(self.res_fc, nn.Linear):
+            nn.init.xavier_normal_(self.res_fc.weight, gain=gain)
```

Comment:
If `bias=True` in module initialization, you will need to reinitialize the bias as well.